Fix UTC dates

sirromdev opened the pull request.

sirromdev commented on src/utils/constants.js:
```diff
@@ -28,9 +28,10 @@ export const PollTxState = {
+export const POLL_DEFAULT_START = getUTC(now);
+export const POLL_DEFAULT_END = getUTC(
+  new Date(now.getTime() + 7 * 24 * 60 * 60 * 1000)
 );
```
Wrapping `POLL_DEFAULT_START` and `POLL_DEFAULT_END` with `getUTC` as above will create Date objects with the correct UTC time (floored to the minute). One caveat is that it will default to use the locale for wherever the user is located and so calling `POLL_DEFAULT_START.toUTCString()` will actually return a time equal to the users timezone offset to UTC less UTC itself. Calling `POLL_DEFAULT_START.toString()` will return the time as UTC.

jparklev commented:
@sirromdev i think the other issue is that the date time picker, while being inited w/ `POLL_DEFAULT_START/END` in utc, is still interpreting times as local [here](https://github.com/makerdao/governance-dashboard/blob/00a7e9af791ab3736dd8a74efbd2dc133569cce9/src/components/CreatePoll/CreatePollTime.js#L47). This results in the poll creation tx being sent w/ timestamps representing what's in the date time picker as if it were the local timezone.

sirromdev commented:
> @sirromdev i think the other issue is that the date time picker, while being inited w/ `POLL_DEFAULT_START/END` in utc, is still interpreting times as local [here](https://github.com/makerdao/governance-dashboard/blob/00a7e9af791ab3736dd8a74efbd2dc133569cce9/src/components/CreatePoll/CreatePollTime.js#L47). This results in the poll creation tx being sent w/ timestamps representing what's in the date time picker as if it were the local timezone.

Yes, we are encumbered by the date time picker. I've modified it this pr to display local and UTC times by only storing unix timestamps in the `start` and `end` state variables. I have also added a delay of half an hour as the contract will override the poll starting time if it is instantiated at the UI load time.

levity commented:
I notice this PR has been open for a while; what should happen here?
